lib/components/ActivityIndicator.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+    <label v-show="busy">
+        Loading label or a spinner gif...
```

Comment:
This template should either be a slot or a fixed svg spinner

Reply:
I will replace it with a slot where the users can display custom icons or gifs.